core/domain/opportunity_domain.py

```diff
@@ -19,22 +19,67 @@
 from __future__ import absolute_import
 from __future__ import unicode_literals
 
-from core import python_utils
 from core import utils
 from core.constants import constants
 
+from typing import Dict, List
+from typing_extensions import TypedDict
+
+
+class PartialExplorationOpportunitySummaryDict(TypedDict):
+    """A dictionary representing partial fields of
+    ExplorationOpportunitySummary object.
+    """
+
+    id: str
+    topic_name: str
+    story_title: str
+    chapter_title: str
+    content_count: int
+    translation_counts: Dict[str, int]
+    translation_in_review_counts: Dict[str, int]
+
+
+class ExplorationOpportunitySummaryDict(
+    PartialExplorationOpportunitySummaryDict
+):
+    """A dictionary representing ExplorationOpportunitySummary object."""
+
+    topic_id: str
```

Comment:
```suggestion
    """A dictionary representing ExplorationOpportunitySummary object."""
    topic_id: str
```

Please try to explain when this dict is used and how it differs from the partial dict, in the docstring.

Reply:
Done. I'm not sure if I clarified it correctly. I only wrote that this dict gets the required fields to represent 
an opportunity from **PartialExplorationOpportunitySummaryDict**.

Reply:
@vojtechjelinek the same thing happens here When I delete the newline between the doctring and the rest of the class, the Linting Checks manager raises an error` W9030: Missing single newline below class docstring.`